src/components/KSelect/KSelect.vue

```diff
@@ -2,6 +2,7 @@
   <div
     :style="widthStyle"
     class="k-select"
+    :class="[$attrs.class]"
```

Comment:
Do we need to do a `modifiedAttrs` computed like you've done in the other files and adjust the `v-bind` on the `KInput` or `KButton` down below?

Reply:
Yep, pushed.